### DISCUSSION TITLE
Questions and answers

### DISCUSSION THREAD

**Discussion by loftwah** [dailydotdev/daily#534] [Feature Requests]
This is in no way just a quick feature to implement but you could potentially compete with Stack Overflow if you added a tab to a questions section, or however it would best be implemented. Chances are it is too big of a change but I like to share ideas in case they happen to be of value.

**Comment by nimrodkra** [comment]
Hi @loftwah that's ain interesting idea, but at the moment we're pursuing other directions in our product roadmap. We'll keep that in mind for a later stage when it might become relevant again.

**Reply by loftwah** [reply]
It would be a nice problem to have :)
